pennylane_forest/qvm.py

```diff
@@ -71,13 +73,24 @@ class QVMDevice(ForestDevice):
     observables = {"PauliX", "PauliY", "PauliZ", "Identity", "Hadamard", "Hermitian"}
 
     def __init__(self, device, *, shots=1024, noisy=False, **kwargs):
-        self._eigs = {}
 
         if shots <= 0:
             raise ValueError("Number of shots must be a positive integer.")
 
         # ignore any 'wires' keyword argument passed to the device
         kwargs.pop("wires", None)
+        analytic = kwargs.get("analytic", False)
+        timeout = kwargs.pop("timeout", None)
+
+        self.parametric_compilation = kwargs.get("parametric_compilation", True)
+
+        if self.parametric_compilation:
+            self._lookup_table = {}
+            self._parameter_map = {}
+            self._parameter_reference_map = {}
```

Comment:
It would be nice to have attribute docstrings describing each attribute, as well as the type of the keys/values in the dictionary